test/com/ableton/JenkinsMocks.groovy

```diff
@@ -17,4 +17,74 @@ class JenkinsMocks {
       return System.properties['user.dir']
     }
   }
+
+  /**
+   * Simple container for holding mock script output.
+   * @see JenkinsMocks#addShMock
+   */
+  class MockScriptOutput {
+    String stdout
+    int exitValue
+
+    MockScriptOutput(String stdout, int exitValue) {
+      this.stdout = stdout
+      this.exitValue = exitValue
+    }
+  }
+
+  /** Holds configured mock output values for the `sh` command. */
+  static Map<String, MockScriptOutput> mockScripts = [:]
+
+  /**
+   * Configure mock output for the `sh` command. This function should be called before
+   * attempting to call JenkinsMocks.sh.
```

Comment:
Please put backticks around `JenkinsMocks.sh` (and maybe also add `()` after `sh`).

Reply:
Done.